freedomofpress/securedrop
# Remove change codename

## Status

Work in progress

## Description of Changes

Fixes https://github.com/freedomofpress/securedrop/issues/4304

Changes proposed in this pull request:

* Remove codename change feature from journalist interface

## Testing

* Make sure nothing left in the docs around changing codenames
* Visit the journalist interface and make sure you can't change the journalist designation of a source
* Make sure the route for changing codename no longer exists

### If you made changes to the server application code:


### If you made changes to documentation:


### If you added or updated a code dependency:

Choose one of the following:
